mods/ts/rules/nod-structures.yaml

```diff
@@ -277,6 +277,34 @@ NATECH:
 	SelectionDecorations:
 		VisualBounds: 86, 58, 0, -4
 
+NATMPL:
+	Inherits: ^Building
+	Buildable:
+		Queue: Building
+		BuildPaletteOrder: 200
+		Prerequisites: natech, ~structures.nod
+	ProvidesPrerequisite@buildingname:
+	Valued:
+		Cost: 2000
+	Tooltip:
+		Name: Temple of Nod
+		Description: Provides access to advanced Nod technologies.
+	Building:
+		Footprint: xxxx xxxx xxxx
+		Dimensions: 4,3
```

Comment:
I was trying to be helpful by commenting these issues at the appropriate trait parts of our yaml.  If you'd prefer I'll make future comments at the PR level and let you hunt through our definitions yourself.